src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.html

```diff
@@ -0,0 +1,78 @@
+<button
+  [matMenuTriggerFor]="multiSelectMenu"
+  aria-label="more scenario Options"
+  class="dropdown-button"
+  [disabled]="disabled"
+  (click)="handleFilterClick()"
+  [ngClass]="{ 'active-filter': hasSelections(), disabled: disabled }">
+  <mat-icon *ngIf="leadingIcon"></mat-icon>
```

Comment:
which icon are we showing here? I think you need to provide the name of it no?

Reply:
oops, yes, sorry for the cruft in this PR